internal/remote/skr_context.go

```diff
@@ -170,8 +170,11 @@ func (s *SkrContext) SynchronizeKymaMetadata(ctx context.Context, kcpKyma, skrKy
 		return nil
 	}
 
```

Comment:
then you need to consider increasing the test coverage of this `SynchronizeKymaMetadata` since you introduced new logic.

Reply:
Added two things:
- unit tests for the newly added `syncBTPRelatedLabels()` function
- extended unit tests for `SynchronizeKymaMetadata()` function